app/grandchallenge/algorithms/views.py

```diff
@@ -267,6 +268,7 @@ def get_context_data(self, **kwargs):
                 "pending_permission_requests": pending_permission_requests,
                 "algorithm_perms": get_perms(self.request.user, self.object),
                 "best_evaluation_per_phase": self.best_evaluation_per_phase,
+                "gpu_type_choices": GPUTypeChoices,
```

Comment:
It should already be available on `object.GPUTypeChoices`

Reply:
It's not. `GPUTypeChoices` is a separate class outside of the Algorithm model.